service/topology_coordinator.cc

```diff
@@ -1113,6 +1113,17 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
         }
     }
 
+    bool has_tablet_transitions() {
+        auto tm = get_token_metadata_ptr();
```

Comment:
nit: has_tablet_transitions() can be made const qualified.

Reply:
Done.